homeassistant/components/zwave_js/services.py

```diff
@@ -529,8 +529,15 @@ def process_results(
             for node_or_endpoint, result in get_valid_responses_from_results(
                 nodes_or_endpoints_list, _results
             ):
-                zwave_value = result[0]
-                cmd_status = result[1]
+                if value_size is None:
+                    # async_set_config_parameter still returns (Value, SetConfigParameterResult)
+                    zwave_value = result[0]
+                    cmd_status = result[1]
+                else:
+                    # async_set_raw_config_parameter_value now returns just SetConfigParameterResult
+                    cmd_status = result
+                    zwave_value = f"parameter {property_or_property_name}"
```

Comment:
Please break out the service change to a separate PR. We don't want to combine fixes and features or changes to existing features with new features.